js/util/mapbox.js

```diff
@@ -2,78 +2,102 @@
 
 var config = require('./config');
 var browser = require('./browser');
+var url = require('url');
+var querystring = require('querystring');
```

Comment:
If we're going to include `url` and `querystring` in our browserify bundle, it would be wise to use them everywhere in this file instead of string manipulation primitives and regexes. Doing so would be a big :+1: for readability and robustness.